crudops/main.go

```diff
@@ -0,0 +1,181 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"log"
+	"os"
+
+	"github.com/joho/godotenv"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	
+)
+
+func main() {
+	 if err := godotenv.Load(); err != nil {
+	 	log.Println("No .env file found")
```

Comment:
maintain proper logs . fmt is not recommended  godotenv.Load()  if you dont have your env file there is no point of going forward. application is depended on .env. try to maintain log levels Info,error,warn so on..